.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,16 @@
+name: E2E
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: 1.18
```

Comment:
1.19 since it's available?

Reply:
Right, I've updated all the presubmits to 1.19 with this PR now.